cardano-sync/src/Cardano/Sync/Types.hs

```diff
@@ -66,3 +67,16 @@ newtype BlockId = BlockId Int
 -- @Word64@ is valid as well.
 newtype MetaId = MetaId Int
     deriving (Eq, Show)
```

Comment:
What are `MetaId` and `BlockId` . Are they used anywhere else in this repo? I am pretty sure they are not because the names clash with identifiers that are generated by Persisitent in `cardano-db/src/Cardano/Db/Schema.hs`.

Reply:
Same applies to `Block` in this same file.

Reply:
If I remove `BlockId`, `Meta` and `Block`, everything still compiles. So why are they here?

Reply:
They were supposed to be here and used by some other functions in the `CardanoDataLayer`. For now, they are not used anywhere so if you want, you can drop them.